src/test/resources/alma/updateAlmaTestFiles.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# author: dr0i
+# date: 2021-02-23
+
+# lists all json test files, get the hbz id from their names, lookup
+# to get the marc xml sources and builds a new test file archive
+
+# set head of xml file
+echo '<?xml version="1.0" encoding="UTF-8"?>
+<collection>' > almaMarcXmlTestFiles
+
+# get all JSON test files; filter names from then
+for hbzId in $(ls *.json|cut -d '/' -f2|cut -d '.' -f1 ); do
+	echo $hbzId
+	# lookup and filter alma url
+	almaXmlUrl=$(curl https://indexes.devel.digibib.net/export/$hbzId |jq .|grep -A2 -B2 '"type": "alma"'|grep url|cut -d '"' -f4)
```

Comment:
This only works if `$hbzId` is ambiguous; in this case HTTP 300 is returned with a JSON response listing the possible alternatives.

Unique IDs, on the other hand, redirect to the target record; in addition, there's currently no way to restrict the lookup to Alma resources.

Reply:
You are correct, and the script lacks many good things. But atm it's good and working fine - we only are interested in ambiguous data (i.e. data with an Aleph URL _and_ and Alma URL - because we are interested in the Alma Marc sources).
Script will definitely to be improved in the future, as soon as demand shows up.

Reply:
... and thx for the lookup API - it's doing very well :)